mmv1/products/filestore/Instance.yaml

```diff
@@ -73,6 +74,15 @@ examples:
       instance_name: 'test-instance'
     # https://github.com/GoogleCloudPlatform/magic-modules/pull/5875#discussion_r844285335
     exclude_test: true
+virtual_fields:
+  - name: 'replicaAction'
```

Comment:
Is "action" an associated term for filestore instance replicas? If not, I think I'd opt for something with `state`, to align with prior art like https://github.com/GoogleCloudPlatform/magic-modules/blob/main/mmv1/products/colab/Runtime.yaml#L65

maybe `replica_desired_state` or something?

Reply:
Ya the exception is to have this expanded in the future to do more then just pause and resume